CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
   - Throw `IllegalArgumentException` instead of `IllegalStateException` when trying to update workflow instance state to an invalid value.
   - Throw `IllegalArugmentException` instead of `RuntimeException` when trying to insert workflow instance with unknown type or with a state that is not a start state.
   - Make `StateVariableTooLongException` extend `IllegalArgumentException` instead of `RuntimeException`.
+  - Fix sql deadlocks in polling for PostgreSQL with skip locked
```

Comment:
Seems like this is done for all databases? sql -> SQL

Reply:
Other databases had a completely different fix already few releases ago. This code path is only for pg

Reply:
Why there are changes in SqlServerDatabaseConfiguration then?

Reply:
The SQL server does support skip locked, and it uses it now in archiving code. But there should never have been any deadlocks there (or at least the archived rows should be so old as to not conflict with engine). The only database that uses the `select * from workflow where id in (update set executor where executor is null returning id)` is postgresql. it is the only sql variant that currently returns true for `hasUpdateReturning`. Technically at least ms sqlserver supports it too, but syntax is very different with `OUTPUT` so I have not taken it into use yet.